### PR TITLE
chore: adding telemetry to backlinks panel

### PR DESCRIPTION
**Pull request by jonathanyeung**
## chore: adding telemetry to backlinks panel

Adding telemetry to Backlinks Panel.  

EDIT: Airtable Updated.

### CODE REVIEW

File: packages/plugin-core/src/workspace.ts [hunk 1]
```diff
@@ -591,12 +592,29 @@ export class DendronExtension implements IDendronExtension {
...
       }
     );
+
+    backlinkTreeView.onDidExpandElement(() => {
+      AnalyticsUtils.track(VSCodeEvents.BacklinksPanelUsed, {
+        type: "ExpandElement",
+      });
+    });
+
+    backlinkTreeView.onDidChangeVisibility(() => {
```
Comment (kevinslin) [inline packages/plugin-core/src/workspace.ts:602]: does this fire on both visible and hide events?

File: packages/plugin-core/src/workspace.ts [hunk 2]
```diff
@@ -591,12 +592,29 @@ export class DendronExtension implements IDendronExtension {
...
+      });
+    });
+
     this.backlinksDataProvider = backlinksTreeDataProvider;
     this.context.subscriptions.push(backlinksTreeDataProvider);
 
     vscode.commands.registerCommand(
       DENDRON_COMMANDS.BACKLINK_SORT_BY_LAST_UPDATED.key,
       sentryReportingCallback(() => {
+        AnalyticsUtils.track(VSCodeEvents.BacklinksPanelUsed, {
```
Comment (kevinslin) [inline packages/plugin-core/src/workspace.ts:614]: can we also report on what the value was set to?

File: packages/plugin-core/src/constants.ts [hunk 3]
```diff
@@ -498,6 +498,10 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: DendronContext.PLUGIN_ACTIVE,
   },
+  GOTO_BACKLINK: {
```
Comment (kevinslin) [inline packages/plugin-core/src/constants.ts:501]: can we hide this from command palette?

**Review by kevinslin — CHANGES_REQUESTED** [review]

File: packages/plugin-core/src/workspace.ts (hunk 1 above)
Comment (jonathanyeung) [reply packages/plugin-core/src/workspace.ts:602]: updated event name + added a state flag.

File: packages/plugin-core/src/constants.ts (hunk 3 above)
Comment (jonathanyeung) [reply packages/plugin-core/src/constants.ts:501]: good catch, forgot how the commands manifest worked haha

**Review by SeriousBug — APPROVED** [review]

**Pull request merged**